common4j/src/main/com/microsoft/identity/common/java/commands/AcquirePrtSsoTokenResult.java

```diff
@@ -0,0 +1,92 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.java.commands;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Map;
+
+import edu.umd.cs.findbugs.annotations.Nullable;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.Accessors;
+
+/**
+ * A DTO for the results from an AcquirePrtSsoToken request.
+ */
+@Builder
+@Getter
+@Accessors(prefix = "m")
+public class AcquirePrtSsoTokenResult {
+
+    /**
+     * The home account id.
+     */
+    @SerializedName("homeAccountId")
+    private final String mHomeAccountId;
+    /**
+     * The local account id.
+     */
+    @SerializedName("localAccountId")
+    private final String mLocalAccountId;
+    /**
+     * The account name.
+     */
+    @SerializedName("accountName")
+    private final String mAccountName;
```

Comment:
should these be replaced by `IAccountRecord`? I recall you made a DTO for that in this PR

Reply:
It was removed based on feedback.